include/pmacc/meta/conversion/SeqToMap.hpp

```diff
@@ -30,10 +30,8 @@ namespace pmacc
     /** convert a list to a map
      *
      * @tparam T_List an mp_list.
-     * @tparam T_UnaryOperator unary operator to translate type from the sequence
+     * @tparam T_MakePairUnaryOperator unary operator to translate type from the sequence
```

Comment:
This is not necessary for this pull request, but this change is also too small for a separate pull request. Thus, this is fine with me.

Reply:
Yes searched for accessor and deleted the line bellow and catched the wrong name.